deploy/monitoring_reader_role.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: monitoring-reader
```

Comment:
if this is used by MUO only, should we give it a MUO related name? to not confuse others as it will be created in openshift-monitoring project

Reply:
+1 to this.

Reply:
Thanks for your review.
Added **muo** prefix to the role name in openshift-monitoring and openshift-config for monitoring-reader and pull secret reader respectively.